CGCharter-1727386911.html

```diff
@@ -62,28 +62,59 @@ <h2 id="goals">
         Incubate new proposals which build on or complement the Social Web WG recommendations.
       </li>
     </ul>
-  
+
     <h2 id="scope-of-work">
       Scope of Work
     </h2>
-    <p class="remove">
-      {TBD: Describe topics that are in scope. For specifications that the CLA
-      patent section applies to, it is helpful to describe the scope in a way
-      that it is clear what types of technologies will be defined in
-      specifications, as opposed to adoption by reference or underlying
-      technology not defined in the proposed spec. Key use cases are often
-      helpful in describing scope. If no specifications will be defined in the
-      group that the CLA patent section applies to, the charter should clearly
-      state that. A clear scope is particularly important where patent
-      licensing obligations may apply.}
+    <p>
+      The group will maintain errata and editor&apos;s drafts for the following recommendations from the Social Web Working Group:
```

Comment:
I think editor's drafts are in general a WG thing, and should not be in scope of the CG, as is usually the case for WG TRs.

I've asked several long time w3c members what happens to a WG TR's "Editor" role after a WG for the document ends, and they all say that the WG work item Editor role ends.

I included a suggestion here because the rest of the change seems like something I'd +1
 
```suggestion
      The group will maintain errata for the following recommendations from the Social Web Working Group:
```

Reply:
Seems reasonable to me (especially if we spell it out explicitly, re better reading experience)